container/experimental/rbe-debian9/BUILD

```diff
@@ -68,7 +68,7 @@ language_tool_layer(
     packages = [
         "libstdc++-6-dev",
     ],
-    tars = ["//third_party/clang:tar"],
+    tars = ["//third_party/clang:debian8_tar"],
```

Comment:
why are we using the debian8 tar for debian9 container?

Reply:
This is because there are only debian8 / ubuntu16_04's tars, no debian9's:
``` bash
$ gsutil ls gs://clang-builds-stable
gs://clang-builds-stable/clang-debian8/
gs://clang-builds-stable/clang-ubuntu16_04/
```

I will follow up on this, on the other hand since clang has almost no dependencies and the binary produced in debian8 totally works in debian9.

Reply:
Can you add a comment?

Reply:
Done. TODO added.